### ISSUE TITLE
LogWarnings/Errors: PFRecoTauDiscriminationAgainstMuon2::discriminate ... has area = 0

### ISSUE THREAD

**Issue by slava77** [cms-sw/cmssw#17574]
this needs to be understood and demoted or addressed otherwise.
The warning appears in standard 136.731 workflow in 90X now.
it has been seen since at least CMSSW_8_1_X_2016-10-03-2300 or 810pre13

e.g. 
```
Begin processing the 62nd record. Run 274199, Event 40119811, LumiSection 21 at 18-Feb-2017 20:29:40.629 CET
%MSG-w PFRecoTauDiscriminationAgainstMuon2::discriminate:   PFRecoTauDiscriminationAgainstMuon2:hpsPFTauDiscriminationByLooseMuonRejection3Boosted  18-Feb-2017 20:29:46 CET Run: 274199 Event: 401198
11
Jet associated to Tau: Pt = 98.4086, eta = -2.4755, phi = 1.65009 has area = 0 !!
%MSG
%MSG-w PFRecoTauDiscriminationAgainstMuon2::discriminate:   PFRecoTauDiscriminationAgainstMuon2:hpsPFTauDiscriminationByTightMuonRejection3Boosted  18-Feb-2017 20:29:46 CET Run: 274199 Event: 40119811
Jet associated to Tau: Pt = 98.4086, eta = -2.4755, phi = 1.65009 has area = 0 !!
%MSG
```

@davidlange6 @roger-wolf

**Comment by davidlange6** [comment]
assign reconstruction

**Comment by slava77** [comment]
fixed in #17620
